### PR TITLE
Added FAQ on mounting volumes to VMs

### LINKED ISSUE #23: We need more info on mouting volumes to VMs
Lima & WSL both pre-mount volumes, but someone asked on slack:

Question was: 
```
Is filesharing working same like Docker Desktop?  I am using rancher desktop and i would like volume my local directiores? Do I have to configure anything on rancher desktop?
```

### PR DESCRIPTION
**Pull request by jtravee**
Signed-off-by: Jennifer Travinski <jennifer.travinski@suse.com>

Closes #23

### CODE REVIEW

**Review by ericpromislow — CHANGES_REQUESTED** [review]
I only see `/Users/$USER` mounted to the VM. I'm not sure about the other folders -- could you consult with Jan or Mark?

**Comment by jandubois** [comment]
> I only see `/Users/$USER` mounted to the VM. I'm not sure about the other folders

Yes, only `/Users/$USER` and `/tmp/rancher-desktop` are currently shared. The list in this PR is from Docker Desktop, not Rancher Desktop. We should extend the list of mounted directories, but not for this release anymore.

Also, I don't know if we want to link to Docker Desktop documentation from the Rancher Desktop FAQ; the link target could change at any time.

File: docs/faq.md [hunk 1]
```diff
@@ -67,6 +67,10 @@ kubectl port-forward -n kube-system $(kubectl -n kube-system get pods --selector
 ```
 Visit [http://127.0.0.1:9000/dashboard/](http://127.0.0.1:9000/dashboard/) in your browser to view the Traefik dashboard.
 
+**Q: Does file sharing work similarly to Docker Desktop? Do I have to do any additional configuration to mount volumes to VMs?**
+
+**A:** Currently, the following directories are shared by default: `/Users/$USER` on macOS, `/home/$USER` on Linux, and `/tmp/rancher-desktop` on both. For Windows, these files are automatically shared via WSL2. We will update the community as more mounted directories are added to this list.
```
Comment (jandubois) [inline docs/faq.md:72]:
```suggestion
**A:** Currently, the following directories are shared by default: `/Users/$USER` on macOS, `/home/$USER` on Linux, and `/tmp/rancher-desktop` on both. For Windows, all files are automatically shared via WSL2.
```

"these files" refers to the files referred to before, which are macOS and Linux specific. So I think "all files" is better.

I think it is implied that we will update the document as our product changes; it is not specific to this particular question.

Comment (jtravee) [reply docs/faq.md]: Committed as suggested, thanks!

**Review by jandubois — APPROVED** [review]

**Pull request merged**